nemo_text_processing/inverse_text_normalization/en/data/time/time_zone_cased.tsv

```diff
@@ -0,0 +1,14 @@
+cst	c s t
+cet	c e t
+pst	p s t
+est	e s t
+pt	p t
+et	e t
+gmt	g m t
+CST	C S T
```

Comment:
what about adding just the cased lines here from line 8-end , in the code for `cased ` use union of regular and _cased.tsv